griptape/drivers/prompt_model/bedrock_llama_prompt_model_driver.py

```diff
@@ -57,22 +57,25 @@ def prompt_stack_to_model_input(self, prompt_stack: PromptStack) -> str:
 
             if first.is_system():
                 prompt_lines.append(f"<s>[INST] <<SYS>>\n{first.content}\n<</SYS>>\n\n")
-                if second and second.is_user():
-                    prompt_lines.append(f"{second.content} [/INST]")
-                else:
-                    raise Exception("System input must be followed by user input.")
+                if second:
+                    if second.is_user():
+                        prompt_lines.append(f"{second.content} [/INST]")
+                    else:
+                        raise Exception("System input must be followed by user input.")
```

Comment:
Doesn't the new nesting break this? If second is None the system input isn't followed by user input but we don't raise the exception. Or perhaps is it that `second` is optional and if it's present it must be user input?

Reply:
If it is indeed optional, it might be a good idea to add a type hint on line 56 to indicate that, so the code is easier to reason about.